docs/ref/django-admin.txt

```diff
@@ -1066,7 +1066,15 @@ Mails the email addresses specified in :setting:`ADMINS` using
 Starts the Python interactive interpreter.
 
 All models from installed apps are automatically imported into the shell
-environment. Models from apps listed earlier in :setting:`INSTALLED_APPS` take
+environment. In addition, the following utility objects are now imported by 
+default:
+
+* ``from django.conf import settings``
+* ``from django.db import connection, reset_queries, models``
+* ``from django.db.models import functions``
+* ``from django.utils import timezone``
+
+Models from apps listed earlier in :setting:`INSTALLED_APPS` take
 precedence. For a ``--verbosity`` of 2 or higher, the automatically imported
```

Comment:
- Avoid saying "now", instread you need a `.. versionchanged:: 6.0` note
- Move "Models from apps listed earlier in :setting:`INSTALLED_APPS` take precedence." to be next to talking about models
- (possibly) do we think these should be listed explicitly or should be say something generic like "as well as some common utilities." and folks can check with `--verbosity=2`?

Reply:
> * Avoid saying "now", instread you need a `.. versionchanged:: 6.0` note

Do you think that in the `.. versionchanged:: 6.0` note we should list explicitly the objects imported or not?